src/js/tabs/pid_tuning.js

```diff
@@ -2228,6 +2252,14 @@ pid_tuning.initialize = function (callback) {
 
             MSP.promise(MSPCodes.MSP_SET_PID, mspHelper.crunch(MSPCodes.MSP_SET_PID))
             .then(() => MSP.promise(MSPCodes.MSP_SET_PID_ADVANCED, mspHelper.crunch(MSPCodes.MSP_SET_PID_ADVANCED)))
+            .then(() => semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_45)
+                        ? MSP.promise(MSPCodes.MSP2_SET_TEXT,
```

Comment:
Space (no tabs) / indentation of 4 spaces please.

Reply:
Done.